src/main/java/com/cloudbees/jenkins/support/impl/AboutJenkins.java

```diff
@@ -501,6 +501,7 @@ private static String humanReadableSize(long size) {
             out.println("---------------");
             out.println();
             out.println("  * Version: `" + Markdown.escapeBacktick(Jenkins.VERSION) + "`");
+            out.println("  * Instance ID: `" + Markdown.escapeBacktick(Jenkins.get().getLegacyInstanceId()) + "`");
```

Comment:
The ticket speaks about the non-legacy one too?

Reply:
That's my understanding as well. See:

- https://wiki.jenkins.io/display/JENKINS/Instance+Identity
- https://github.com/jenkinsci/jenkins/blob/01d7f443c9e3178da7222e1dff3fc8cf82bc6943/core/src/main/java/jenkins/model/identity/IdentityRootAction.java#L39-L42

Reply:
I added the one from the Identity Module.